mslib/msui/mpl_pathinteractor.py

```diff
@@ -977,6 +977,7 @@ def button_release_insert_callback(self, event):
         y = event.ydata
         wpm = self.waypoints_model
         flightlevel = float(pressure2flightlevel(y * units.Pa).magnitude)
+        flightlevel = 5.0 * round(flightlevel / 5)
```

Comment:
I am not sure if I understand this again in a year , or what happen when someone refactors this

would you please so kind and add a comment above this line.